<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentSupervise</class>
 <widget class="QDialog" name="CommentSupervise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>评论管理</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>175</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/back.png</normaloff>Resources/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>175</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/next.png</normaloff>Resources/next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbComment">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>175</y>
     <width>425</width>
     <height>450</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="teContent">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>325</width>
      <height>225</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbStar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblStar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评价星级：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbls1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbls2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbls3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbls5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbls4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblContent">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>文字内容：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbProperty">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>410</width>
     <height>400</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDateEdit" name="deDate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>275</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd</string>
    </property>
    <property name="timeSpec">
     <enum>Qt::TimeZone</enum>
    </property>
    <property name="date">
     <date>
      <year>2017</year>
      <month>9</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDestId">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSrcId">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>125</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leComId">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblComId">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评论ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSrcId">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>125</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评论者ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDestId">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评论对象ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>275</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评论日期：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评论标题：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTitle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>175</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>保存改动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUndo">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>175</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销改动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>175</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/subtract.png</normaloff>Resources/subtract.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>175</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/add.png</normaloff>Resources/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblCommSVBGN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>650</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblCommSVBGN</zorder>
  <zorder>btnBack</zorder>
  <zorder>btnNext</zorder>
  <zorder>gbComment</zorder>
  <zorder>gbProperty</zorder>
  <zorder>btnSave</zorder>
  <zorder>btnUndo</zorder>
  <zorder>btnDel</zorder>
  <zorder>btnAdd</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sbStar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>SetStars(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDel</sender>
   <signal>clicked()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnSaveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUndo</sender>
   <signal>clicked()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnUndoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnAddClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBack</sender>
   <signal>clicked()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>BackOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>NextOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leComId</sender>
   <signal>textEdited(QString)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSrcId</sender>
   <signal>textEdited(QString)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDestId</sender>
   <signal>textEdited(QString)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTitle</sender>
   <signal>textEdited(QString)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deDate</sender>
   <signal>userDateChanged(QDate)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbStar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teContent</sender>
   <signal>textChanged()</signal>
   <receiver>CommentSupervise</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>885</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetStars(int)</slot>
  <slot>OnDelClick()</slot>
  <slot>OnSaveClick()</slot>
  <slot>OnUndoClick()</slot>
  <slot>OnAddClick()</slot>
  <slot>BackOne()</slot>
  <slot>NextOne()</slot>
  <slot>OnDelta()</slot>
 </slots>
</ui>
